src/account/accountReducers.js

```diff
@@ -90,6 +91,12 @@ export default (state: AccountState = initialState, action: Action) => {
       newState.splice(action.index, 1);
       return newState;
     }
+    case SAVE_GCM_TOKEN: {
+      return [
+        { ...state[0], gcmToken: action.gcmToken },
+        state.splice(1, state.length - 1)
```

Comment:
`splice` is incorrect. It mutates the list. You might mean `slice`.
If you had tests this would happen less frequently ;)